packages/cli/src/commands/generate.ts

```diff
@@ -0,0 +1,150 @@
+import Command from '../command'
+import {flags} from '@oclif/command'
+import * as fs from 'fs'
+import * as path from 'path'
+import chalk from 'chalk'
+import enquirer from 'enquirer'
+const debug = require('debug')('blitz:generate')
+
+enum ResourceType {
+  Resource = 'resource',
+  Model = 'model',
```

Comment:
We don't currently have model files, so just leave this out for now

Reply:
I was figuring that model would add the model to the prisma file, generate a query + mutation, and then run `prisma migrate`. Just a shorthand for adding data to the `db` without generating a UI for it